components/component.go

```diff
@@ -195,3 +201,34 @@ func (c *Component) UpdatePrometheusConfig(_ client.Client, logger logr.Logger,
 
 	return err
 }
+
+func GetReleaseVersion(defaultManifestPath string, componentName string) status.ComponentStatus {
```

Comment:
I will not block the PR, but with that the functions looks like `GetComponentStatus` and belongs status package.

Reply:
this is done